### PR TITLE
Temporary fix for Sentient Anomalies

### PR DESCRIPTION
**Pull request by EricSihaoLin**
**Summary:**

Stop-gap fix for Sentient Anomalies changes recently until a more permanent solution is found.

---
**Fixes issue (include link):**

---
**Mockups, screenshots, evidence:**

image

### CODE REVIEW

File: src/components/panels/SentientOutpostsPanel.vue [hunk 1]
```diff
@@ -29,17 +34,23 @@ import TimeBadge from '@/components/TimeBadge.vue';
...
-import moment from 'moment';
+//import moment from 'moment';
 
 import sentient from '@/assets/img/factions/sentient.svg';
 
 export default {
   name: 'SentientOutpostsPanel',
   props: ['sentientOutposts'],
+  watch: {
+    mission: function() {
```
Comment (EricSihaoLin) [inline src/components/panels/SentientOutpostsPanel.vue:45]: Codacy Issue found: [Insert `·`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5481243)

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5481243&bid=17962830)

File: src/components/panels/SentientOutpostsPanel.vue (hunk 1 above)
Comment (TobiTenno) [reply src/components/panels/SentientOutpostsPanel.vue:45]: fix it and i'll merge

File: src/components/panels/SentientOutpostsPanel.vue [hunk 2]
```diff
@@ -29,17 +34,23 @@ import TimeBadge from '@/components/TimeBadge.vue';
...
+//import moment from 'moment';
 
 import sentient from '@/assets/img/factions/sentient.svg';
 
 export default {
   name: 'SentientOutpostsPanel',
   props: ['sentientOutposts'],
+  watch: {
+    mission: function () {
+      this.active = new Date().getTime();
```
Comment (TobiTenno) [inline src/components/panels/SentientOutpostsPanel.vue:43]: uhhh, let's just use outpost.activation? i can go fix it on the api, this is just kinda jank 🤔

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5481243&bid=17962830)

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5481243&bid=17962830)

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**